src/VisualStudio.Roslyn.SDK/Roslyn.SDK/ProjectTemplates/CSharp/Diagnostic/Test/DiagnosticVerifier.Helper.cs

```diff
@@ -15,10 +15,10 @@ namespace TestHelper
     /// </summary>
     public abstract partial class DiagnosticVerifier
```

Comment:
📝 This is a legacy test class that is no longer recommended for use. The new test library, which we would like to incorporate into the templates, does not include this file at all. See #154 for my initial work on this that stalled a while back.

Reply:
Ok, I see. However this is still what you get when you use the template from VS :-(